### PR TITLE
Refs #22125 - Use react-json-tree for report json rendering

### PR DESCRIPTION
**Pull request by bastilian**
This implements a React component, which is based on [react-jsontree](https://www.npmjs.com/package/react-json-tree), to display JSON of ansible reports nicely in a tree that can be expanded and collapsed at will:

The styling still needs some tweaks to integrate better into Foreman, but it's already better than what is at the moment:

gnome-shell-screenshot-m5ajez

### CODE REVIEW

File: webpack/components/ReportJsonViewer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const theme = {
+  scheme: 'foreman',
+  backgroundColor: 'rgba(0, 0, 0, 0)',
+  base00: 'rgba(0, 0, 0, 0)',
+};
+
+class ReportJsonViewer extends React.Component {
+  render () {
+    console.log(this);
+    return <JSONTree data={this.props.data} hideRoot theme={theme} />
```
Comment (dLobatog) [inline webpack/components/ReportJsonViewer.js]: Is this hiding the root on the module_args? I see the following 

screenshot from 2018-02-26 09-02-57

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "foreman_ansible",
+  "version": "1.0.0",
+  "description": "[![Code Climate](https://codeclimate.com/github/theforeman/foreman_ansible/badges/gpa.svg)](https://codeclimate.com/github/theforeman/foreman_ansible) [![Gem Version](https://badge.fury.io/rb/foreman_ansible.svg)](https://badge.fury.io/rb/foreman_ansible) [![GPL License](https://img.shields.io/github/license/theforeman/foreman_ansible.svg)](https://github.com/theforeman/foreman_ansible/blob/master/LICENSE)",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
```
Comment (dLobatog) [inline package.json]: If you could please add eslint & hound here, it'd be great to ensure the code standards are followed from day 1.

https://github.com/theforeman/foreman_remote_execution/pull/130 for an example of that

**Comment by bastilian** [comment]
@dLobatog added eslint. Should we also enable it in codeclimate?

**Comment by bastilian** [comment]
@dLobatog The test is changed and should now pass. 

I tried to apply CSS, but for some reason neither importing a CSS for the component nor adding a simple stylesheet to precompile assets doesn't include it for some reason. :/ The later is committed in here.

**Comment by bastilian** [comment]
The CSS is no included and works as it should to make the background transparent. \o/

**Comment by bastilian** [comment]
Thank you for reviewing & merging @dLobatog!

**Pull request merged**